packages/dds/tree/src/change-family/changeEncoder.ts

```diff
@@ -4,9 +4,11 @@
  */
 
 import { bufferToString, IsoBuffer } from "@fluidframework/common-utils";
+import { Jsonable } from "@fluidframework/datastore-definitions";
 
 export abstract class ChangeEncoder<TChange> {
     public abstract encodeForJson(formatVersion: number, change: TChange): JsonCompatible;
```

Comment:
I think this PR would be much more valuable if we can delete `JsonCompatible` by doing something like this:

```suggestion
export abstract class ChangeEncoder<TChange, Jsonable<TEncoded>> {
    public abstract encodeForJson(formatVersion: number, change: TChange): TEncoded;
```